lib/templates/bootstrap/textbox.js

```diff
@@ -1,5 +1,6 @@
 var React = require('react');
 var { View, Text, TextInput } = require('react-native');
+import TextField from 'react-native-md-textinput';
```

Comment:
This is an external dependency but you haven't modified the `package.json`. Why exactly do you need the Material Design `TextInput`?